common.py

```diff
@@ -354,48 +354,109 @@ def initialize_theta(theta_shape, possible_theta_mask, zero_theta_mask):
     theta[zero_theta_mask] = 0
     return theta
 
-def split_train_val(num_obs, metadata, tuning_sample_ratio, validation_column=None, val_column_idx=None):
+def create_theta_idx_mask(zero_theta_mask_refit, possible_theta_mask):
```

Comment:
@dawahs  -- i had to put back this function because it seemed like it was getting used elsewhere?